### PR TITLE
Prepare release notes

### PR DESCRIPTION
**Pull request by t-b**

### CODE REVIEW

File: Packages/doc/releasenotes.rst [hunk 1]
```diff
@@ -3,6 +3,404 @@ Release notes
...
+- Make the panel naming more consistent
+- Fix sorting of sweeps in OVS in edge case
+- Enhance accuracy of epoch info in display
+- Fix labnotebook/results name gathering speed for popup menu
+
+Dashboard
+~~~~~~~~~
+
+- Fix calculation of failed sweeps for Rheobase
+- Allow to select multiple disjoint rows
```
Comment (timjarsky) [inline Packages/doc/releasenotes.rst:142]: ...multiple individual rows

File: Packages/doc/releasenotes.rst [hunk 2]
```diff
@@ -3,6 +3,404 @@ Release notes
...
+  window is not brought to the front anymore on update.
+
+
+
+
+AnalysisBrowser
+---------------
+
+- Add on-the-fly upgrade for sweep wave format
+- Fix file management when removing and readding files in edge case
```
Comment (timjarsky) [inline Packages/doc/releasenotes.rst:125]: ....reading.....

File: Packages/doc/releasenotes.rst [hunk 3]
```diff
@@ -3,6 +3,404 @@ Release notes
...
+~~~~~~~~~
+
+- Fix calculation of failed sweeps for Rheobase
+- Allow to select multiple disjoint rows
+- Decorate the analysis function name with the operation mode (``PSQ_DASCale`` only)
+
+DA\_Ephys
+---------
+
+- (Experimental) Add support for Sutter hardware
```
Comment (timjarsky) [inline Packages/doc/releasenotes.rst:148]: ...Sutter DAC hardare?? 

Amp hardware is pending?

File: Packages/doc/releasenotes.rst (hunk 2 above)
Comment (t-b) [reply Packages/doc/releasenotes.rst:125]: I really meant adding again aka readding. Maybe with a dash?

Comment (timjarsky) [reply Packages/doc/releasenotes.rst:125]: how about "restoring" @t-b

**Review by timjarsky — APPROVED** [review]

**Pull request merged**